libraries/azurerm_virtual_networks.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+
+class AzurermVirtualNetworkss < AzurermPluralResource
+  name 'azurerm_virtual_networks'
+  desc 'Verifies settings for Azure Virtual Networks'
+  example <<-EXAMPLE
+    azurerm_virtual_networks(resource_group: 'example') do
+      it{ should exist }
+    end
+  EXAMPLE
+
+  FilterTable.create
+             .add_accessor(:entries)
+             .add_accessor(:where)
+             .add(:exists?) { |obj| !obj.entries.empty? }
+             .add(:names, field: 'name')
+             .connect(self, :table)
```

Comment:
Please use the newer `FilterTable` syntax. See here for an example: https://github.com/inspec/inspec-azure/blob/master/libraries/azurerm_virtual_machines.rb#L14-L18

(This just changed recently, sorry.)